app/policies/course_lesson_policy.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class CourseLessonPolicy < ApplicationPolicy
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def show?
+    true
+  end
+
+  def create?
+    admin_only
+  end
+
+  def edit?
```

Comment:
Don't think we need the edit here, since it's the same as update. Double check ApplicationPolicy, but I'm pretty sure `edit?` just returns `update?` by default, so we can leave it like that